cmd/mycelo/main.go

```diff
@@ -110,6 +114,16 @@ var gethPathFlag = cli.StringFlag{
 	Usage: "Path to geth binary",
 }
 
+var loadTestTPSFlag = cli.IntFlag{
+	Name:  "tps",
+	Usage: "Transactions per second to target in the load test",
+}
+
+var verboseFlag = cli.BoolFlag{
```

Comment:
notice that there's already a `mycelo --verbosity=5 load-bot` option. So might be confusing.

That is added on line 42 `debug.Setup()`

Reply:
Switched to requiring `--verbosity 5` for verbose output